.github/workflows/deploy-app.yml

```diff
@@ -28,7 +28,7 @@ name: Deploy to Amazon ECS
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "main" , "feat/setup-cicd"]
```

Comment:
should we remove the feat/setup-cicd?

Reply:
I was doing some testing for the aws deployment, will not merge this into main.  Will cancel this PR.